docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md

```diff
@@ -26,33 +26,33 @@ Let us tart with the most basic question.
 
 In simple and cheesy terms, GRAKN is a database on steroids that makes it easy to build intelligent applications.
 
-More precisely, it is what we call a [Knowledge Based System](https://en.wikipedia.org/wiki/Knowledge-based_systems): a piece of software that allows to model complex problems, made of several components:
+More precisely, it is what we call a [Knowledge Graphd System](https://en.wikipedia.org/wiki/Knowledge-based_systems): a piece of software that allows to model complex problems, made of several components:
```

Comment:
:goat:

Reply:
Fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f